### ISSUE TITLE
less files available publicly

### ISSUE THREAD

**Issue by markokocic** [keystonejs/keystone#1639]
Every file under `/public/styles/` is available through http when `keystone` is running.

Since `*.less` files could be considered as a part of the source code, could we move them to another  location, e.g. `/templates/styles/` to protect them from being available publicly?

It may be a good idea to keep `/public/` folder only for files that are required during runtime, and keep source files elsewhere.

**Comment by cameronjroe** [comment]
I agree with this. Ideally the gulp build would output dev/production builds of assets to `/public/{js, css, img, etc}` and all the `.less` and asset sources for the client would live behind the src directories.
